File: q7/q7.py
def uniqueCharacters(str):
    for i in range(len(str)):
        for j in range(i + 1, len(str)):
            if(str[i] == str[j]):
                return False
    return True


str = "Jelly Techno"

if(uniqueCharacters(str)):
    print("Text ", str, " termasuk uniq karakter")
else:
    print("Text ", str, " termasuk duplikat karakter")

File: q7/q7.php
<?php

function uniqueCharacter($text)
{
    for ($i = 0; $i < strlen($text); $i++) {
        for ($j = $i + 1; $j < strlen($text); $j++) {
            if ($text[$i] == $text[$j]) {
                return false;
            }
        }
    }
    return true;
}

$data = "Jelly Techno";
if (uniqueCharacter($data)) {
    echo "Text $data termasuk uniq karakter";
} else {
    echo "Text $data termasuk duplikat karakter";
}
